components/feature/prompts/src/main/java/mozilla/components/feature/prompts/file/FilePicker.kt

```diff
@@ -197,6 +198,8 @@ internal class FilePicker(
                 }
             } ?: request.onDismiss()
         }
+
+        captureUri = null
```

Comment:
Could we verify in tests that we are cleaning this value? :)

Reply:
Looked into this but I'm not sure how.
With `mockk` we would have access to this file scoped properties but I don't know if `mockito-kotlin` supports this.
Any pointers are much appreciated.

Reply:
As this property is static (global) I think we only need to change the visibility to `internal` and add the `@VisibleForTesting`, then verify from the test that after `handleFilePickerIntentResult` `captureUri` is `null`

Reply:
Thank you!